drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodModel.kt

```diff
@@ -9,6 +9,10 @@
 package com.adyen.checkout.dropin.ui.paymentmethods
 
 data class PaymentMethodModel(
-    val type: String,
-    val name: String
-)
+    val type: String?,
+    val name: String,
+    val icon: String,
+    val isBorderDisabled: Boolean
```

Comment:
Although it is the exception, I think it's confusing to have a flag about it being disabled.
Could we make it something like `val drawIconBorder: Boolean = true`?